src/Explorer.js

```diff
@@ -48,9 +52,12 @@ import type {
   ObjectValueNode,
   SelectionNode,
   SelectionSetNode,
+  VariableDefinitionNode,
   ValueNode,
 } from 'graphql';
 
+const AvailableFragmentsContext = React.createContext({});
```

Comment:
Can you pass the fragments down instead of using context? We've avoided using context for everything else, the main reason being that we want to support older versions of react, but this api is going to be deprecated in the future.